FreeRTOS/FreeRTOS
# Update PKCS #11 in LTS development branch

Update PKCS #11 in LTS development branch

Description
-----------
This PR syncs the LTS branch with release-candidate branch on AFR

Test Steps
-----------
Run PKCS #11 demo locally

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
